### PR TITLE
Add a formula for the Secrets Manager provider

### PR DESCRIPTION
**Pull request by tabacco**

### CODE REVIEW

File: summon-aws-secrets.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class SummonAwsSecrets < Formula
+  desc "Conjur provider for AWS Secrets Manager"
```
Comment (izgeri) [inline summon-aws-secrets.rb]:
```suggestion
  desc "Summon provider for AWS Secrets Manager"
```

File: summon-aws-secrets.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  version "0.3.0"
+  sha256 "f995961357f82709c78a9850186d6110a6075bc7d61a9eadc50e28b08346c455"
+
+  def install
+    target = lib/"summon"
+    target.install "summon-aws-secrets"
+  end
+
+  test do
+    system lib/"summon"/"summon-aws-secrets", "-V"
```
Comment (izgeri) [inline summon-aws-secrets.rb]: this should be a lowercase v (see the [docs](https://github.com/cyberark/summon-aws-secrets#flags))

File: summon-aws-secrets.rb (hunk 1 above)
Comment (tabacco) [reply summon-aws-secrets.rb]: Heh whoops!

**Comment by tabacco** [comment]
Updated! Sorry for the copypasta and one brain lapse.

**Pull request merged**